api/storyblok-sync/index.js

```diff
@@ -25,6 +25,22 @@ module.exports = ({ config, db }) => {
     }, 404)
   })
 
+  const fetchStories = async () => {
+    console.log('🧱 : Fetching stories!') // eslint-disable-line no-console
+    const languages = [null].concat(config.storyblok.extraLanguages || [])
+    const promises = languages.map(lang => getStories({
+      token: config.storyblok.accessToken
+    }, 1, lang))
+    const result = await Promise.all(promises)
+    const stories = [].concat.apply([], result).map(transformStory(index))
+    return Promise.all(stories.map(story => db.update(story)))
+  }
+
+  // Run once on startup
+  setTimeout(() => {
+    fetchStories()
+  }, 10000) // Let elasticsearch start
```

Comment:
Shouldn't we assume that elastic search is started an available and it's up to the infrastructure setup to make sure elastic search is stared before vue-storefront-api? i.e. using healthcheck in docker or something similar in kubernetes/whatever?

Reply:
I think it's fair to wait 10 seconds either way